add_dax_stocks.sh

```diff
@@ -1,34 +1,34 @@
+./analyser add -u "http://www.onvista.de/aktien/SAP-Aktie-DE0007164600" -d "http://www.finanzen.net/termine/SAP" "SAP.DE"
+./analyser add -u "http://www.onvista.de/aktien/ThyssenKrupp-Aktie-DE0007500001" -d "http://www.finanzen.net/termine/thyssenkrupp" "TKA.DE"
+./analyser add -u "http://www.onvista.de/aktien/Vonovia-Aktie-DE000A1ML7J1" -d "http://www.finanzen.net/termine/Vonovia" "VNA.DE"
+./analyser add -u "http://www.onvista.de/aktien/HeidelbergCement-Aktie-DE0006047004" -d "http://www.finanzen.net/termine/HeidelbergCement" "HEI.DE" 
+./analyser add -u "http://www.onvista.de/aktien/Fresenius-Aktie-DE0005785604" -d "http://www.finanzen.net/termine/Fresenius" "FRE.DE"
+./analyser add -u "http://www.onvista.de/aktien/Fresenius-Medical-Care-Aktie-DE0005785802" -d "http://www.finanzen.net/termine/Fresenius_Medical_Care" "FME.DE"
+./analyser add -u "http://www.onvista.de/aktien/EON-Aktie-DE000ENAG999" -d "http://www.finanzen.net/termine/EON" "EOAN.DE"
+./analyser add -u "http://www.onvista.de/aktien/Deutsche-Telekom-Aktie-DE0005557508" -d "http://www.finanzen.net/termine/Deutsche_Telekom" "DTE.DE"
+./analyser add -u "http://www.onvista.de/aktien/Deutsche-Post-Aktie-DE0005552004" -d "http://www.finanzen.net/termine/Deutsche_Post" "DPW.DE"
+./analyser add -u "http://www.onvista.de/aktien/Lufthansa-Aktie-DE0008232125" -d "http://www.finanzen.net/termine/Lufthansa" "LHA.DE"
+./analyser add -u "http://www.onvista.de/aktien/Daimler-Aktie-DE0007100000" -d "http://www.finanzen.net/termine/Daimler" "DAI.DE"
+./analyser add -u "http://www.onvista.de/aktien/Continental-Aktie-DE0005439004" -d "http://www.finanzen.net/termine/Continental" "CON.DE"
+./analyser add -u "http://www.onvista.de/aktien/Beiersdorf-Aktie-DE0005200000" -d "http://www.finanzen.net/termine/Beiersdorf" "BEI.DE"
+./analyser add -u "http://www.onvista.de/aktien/BMW-Aktie-DE0005190003" -d "http://www.finanzen.net/termine/BMW" "BMW.DE"
+./analyser add -u "http://www.onvista.de/aktien/Bayer-Aktie-DE000BAY0017" -d "http://www.finanzen.net/termine/Bayer" "BAYN.DE"
+./analyser add -u "http://www.onvista.de/aktien/BASF-Aktie-DE000BASF111" -d "http://www.finanzen.net/termine/BASF" "BAS.DE"
+./analyser add -u "http://www.onvista.de/aktien/Adidas-Aktie-DE000A1EWWW0" -d "http://www.finanzen.net/termine/Adidas" "ADS.DE"
 
 # Finanzwerte, benötigen sonderregeln in der Levermann Analyse:
-# ./analyser add -u "http://www.onvista.de/aktien/Muenchener-Rueck-Aktie-DE0008430026" -d "07.02.2017" "MUV2.DE"
-# ./analyser add -u "http://www.onvista.de/aktien/DEUTSCHE-BOERSE-AG-Aktie-DE000A2AA253"
-# ./analyser add -u "http://www.onvista.de/aktien/Deutsche-Bank-Aktie-DE0005140008"
-# ./analyser add -u "http://www.onvista.de/aktien/Commerzbank-Aktie-DE000CBK1001"
-#./analyser add -u "http://www.onvista.de/aktien/Allianz-Aktie-DE0008404005" -d "11.11.2016" "ALV.DE"
+./analyser add -u "http://www.onvista.de/aktien/Muenchener-Rueck-Aktie-DE0008430026" -d "07.02.2017" "MUV2.DE"
```

Comment:
They were commented because the levermann analysis is slightly different for finance stocks. 
For example they only need to have >=10% equity ratio instead of >=25% to get a point in that category.